kernel_install

```diff
@@ -38,9 +38,32 @@ kernel_revision="$(echo "${kernel_ext}" | cut -d '-' -f 3)"
 boot_name="Gentoo ${kernel_version}"
 [ "${kernel_tag}" != 'gentoo' ] && boot_name="${boot_name} ${kernel_tag}"
 
+efi_ro=false
+if mount | grep "^efivarfs" | grep "ro"
+then
+	efi_ro=true
+	efi_mpoint=$(mount | grep "^efivarfs" | cut -d " " -f 3)
+fi
+
+while $efi_ro; do
+	read -p "Allow changes in efivarfs? " yn
+    	case $yn in
+        	[Yy]es)	mount -o remount,rw,relatime efivarfs $efi_mpoint
```

Comment:
`relatime` is by default since a long time (in the man page)
> Since Linux 2.6.30, the kernel defaults to the behavior provided by this option [`relatime`]

Reply:
Done